subprojects/docs/src/docs/userguide/publishing_maven.adoc

```diff
@@ -106,6 +106,23 @@ include::sample[dir="signing/maven-publish/groovy",files="build.gradle[tags=pom-
 include::sample[dir="signing/maven-publish/kotlin",files="build.gradle.kts[tags=pom-customization]"]
 ====
 
+[[publishing_maven:resolved_dependencies]]
+=== Customizing dependencies versions
+
+The set of dependencies and constraints that are added to the POM file will use by default the _declared_ versions from the build file itself.
```

Comment:
-> `By default, the set of dependencies and constraints that are added to the POM file will contain the versions _declared_ in the build file.`